packages/core/src/protocol-interfaces.ts

```diff
@@ -55,7 +55,7 @@ export interface ProtocolClientFactory {
 
 export interface ProtocolServer {
   readonly scheme: string;
-  expose(thing: ExposedThing, tdTemplate?: WoT.ThingDescription): Promise<void>;
+  expose(thing: ExposedThing, tdTemplate?: WoT.ExposedThingInit): Promise<void>;
```

Comment:
Maybe we can use the default pattern instead of undefined: `tdTemplate?: WoT.ExposedThingInit = {}`